Updated Turkish translation of the README file.

karsibali opened the pull request:
Updating the Turkish translation is my first contribution to this project. So, please add my name to the contributers list too.
Thanks...

karsibali committed:
Updated Turkish translation of the README file. So, add Musa Serdar Arslan to the contributers list too.

Roshanjossey commented:
Hi @karsibali, thank you for your contribution. Lemme try to get a review from one of our previous contributors of Turkish translation.

@acgrdumlu could you please do a review on this translation?

karsibali commented:
Well, the Turkish translation of the README file was already there. But, some parts of it was not very good, much of a word-for-word translation in my opinion. So, I changed it just a little bit to make it more understandable in Turkish. I think it was also from a previous version of the README file. So, I changed it to reflect the current version.

yamac-kurtulus reviewed (changes requested):
Hi..
Before seeing this PR, I myself made very similar changes on the translation as well. It looks good to me, albeit a few issues. I found some typos in 8th line, "kolyalaştırmak" Same line, "eğer ... bulunacaksınız" and take a look at line 112, where "yapmış olduğunuz katkıdan kutlama yapın" sounds off to me. 

Also, the readme file has changed a bit, the badges on the top for instance. And the src of the link for git-kraken.md seems to be off.

I really appreciate your effort to translate the terms by the way. I would suggest changing "markup dili olan..." into "işaretleme dili olan." likewise, "cheat-sheet" to "kopya kağıdı" and "johndoe" to "ahmetyilmaz" :). I have no really good candidates for pushing or pull request sadly :/ I hesitantly used "itmek" and "çekme isteği"

Here is my PR by the well, feel free to inspect and use the parts where you liked better. 
 #3598

yamac-kurtulus commented:
@karsibali 
Then I'll make the suggested changes and push. Thanks for your kind words and help.

Roshanjossey commented:
@karsibali, thank you for your suggestions.

Also you people are really good at collaborating. I'm closing this pull request and merging #3598
